velero-plugins/pod/restore.go

```diff
@@ -32,39 +32,44 @@ func (p *RestorePlugin) Execute(input *velero.RestoreItemActionExecuteInput) (*v
 	itemMarshal, _ := json.Marshal(input.Item)
 	json.Unmarshal(itemMarshal, &pod)
 	p.Log.Infof("[pod-restore] pod: %s", pod.Name)
+	
+	if input.Restore.Annotations[common.MigrateCopyPhaseAnnotation] == "stage" {
```

Comment:
We should also probably include the "part-of=openshift-migration" comparison here.

Reply:
Does this look good?

p.Log.Infof("[pod-restore] pod: %s", pod.Name)
	
	if (input.Restore.Annotations[common.MigrateCopyPhaseAnnotation] == "stage") && (input.Restore.Labels["app.kubernetes.io/part-of"] == "openshift-migration"){
		pod.Labels[common.MigratePodStageLabel] = "true"
		pod.Spec.Affinity = nil
	} else {

Reply:
Mostly, yes. A couple minor updates, though:
1) I'd swap the conditions. First check if it's migration (if it's not a migration, no need to check any further)
2) Use consts instead of hard-coded strings for the label name and value.

Reply:
Have done the requested changes